WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -781,25 +781,14 @@ private extension ProductFormViewController {
 
     /// Updates table viewmodel and datasource and attempts to animate cell deletion/insertion.
     ///
-    func reloadLinkedPromoCellAnimated() {
-        let indexPathBeforeReload = findLinkedPromoCellIndexPath()
+    func reloadLinkedPromoCell() {
         tableViewModel = DefaultProductFormTableViewModel(product: viewModel.productModel,
                                                           actionsFactory: viewModel.actionsFactory,
                                                           currency: currency,
                                                           isDescriptionAIEnabled: aiEligibilityChecker.isFeatureEnabled(.description))
-        let indexPathAfterReload = findLinkedPromoCellIndexPath()
```

Comment:
Should we remove `findLinkedPromoCellIndexPath` method as we longer use it?

Reply:
Removed in 79f16a76e7c75f4e5075c6888009ad4495fd3bb5.